www/artists/emmalake/gallery.html

```diff
@@ -22,34 +22,53 @@ <h1>
                     <table border="0" width="75%">
                         <tr>
                             <td width="20%">
-                                <a id="contentlink" href="../amckay/1.jpg"><img id="thumbnail" src="a_mckay.jpeg" alt="no alt text"></a>
+                                Art McKay
+                                <br>
+                                <a id="contentlink" href="https://www.saskartists.ca/artists/amckay/index.html"><img id="thumbnail" src="a_mckay.jpeg" alt="no alt text"></a>
                             </td>
                             <td width="20%">
+                                "Flux"
+                                <br>
                                 <img src="flux.jpeg" alt="no alt text">
                             </td>
                             <td width="20%">
+                                Untitled
+                                <br>
                                 <img src="round_art.jpeg" alt="no alt text">
                             </td>
                             <td width="20%">
+                                His Rectangular art
+                                <br>
+                                </article>
                                 <img src="round_art2.jpeg" alt="no alt text">
                             </td>
                         </tr>
                         <tr>
                             <td width="20%">
-                                <img src="kennethlochhead.jpeg" alt="no alt text">
+                                Kenneth Lochhead
+                                <br>
+                                <a id="contentlink" href="https://www.saskartists.ca/artists/klochhead/index.html"><img src="kennethlochhead.jpeg" alt="no alt text">
```

Comment:
We shouldn't have multiple items with the same id in the same page. IDs are supposed to be unique. Classes can overlap - consider changing contentlink to a class rather than an ID.